validator/validator-main.protoascii

```diff
@@ -3227,6 +3227,26 @@ tags: {  # <amp-pixel>
   }
 }
 
+tags: {
```

Comment:
Please move this to be with the other source tags (4.7.8). They begin on line [1473](https://github.com/ampproject/amphtml/blob/master/validator/validator-main.protoascii#L1473)